### ISSUE TITLE
Rewrite internal coordinate system?

### ISSUE THREAD

**Issue by mathiasbynens** [tibiamaps/tibia-map#9]
Currently the coordinates are represented in the format `x = 124, y = 121` instead of `x = 31744, y = 30976`, meaning we have to convert them to the latter format every time we want to display them. Switching to the latter format internally might simplify things and make it easier to add a “markers” layer later on.

Resources that might be useful:
- http://leafletjs.com/examples/crs-simple/crs-simple.html

**Comment by Bernardoow** [comment]
@mathiasbynens 

Hi, can you give more information about this issue?

**Comment by mathiasbynens** [comment]
For example, see https://github.com/tibiamaps/tibia-map/blob/8178bc02b4c84bb8b5e6bb4233c916b3bd1dd7a8/src/_js/map.js#L119, where we have to convert from the old-style coordinates `x = 124, y = 121` to `x = 31744, y = 30976` by multiplying with `256`.

The same thing happens here: https://github.com/tibiamaps/tibia-map/blob/8178bc02b4c84bb8b5e6bb4233c916b3bd1dd7a8/src/_js/map.js#L252-L256

The old coordinates made sense when the mapper was using a version of the `Automap` files, whose file names followed the old format. But especially now that we’ve switched to use `minimap` files, the new format would make more sense.

Did that help clarify what I mean? Please feel free to ask more questions!

**Comment by Bernardoow** [comment]
@mathiasbynens 

I think I understand what yout want. I don't know if is possibile. I'm a newbie with leftletjs . But What I think, this issue is complicate.

I checked the code and leftlet use  projection and unprojection to do from/to.

See test I did:

https://github.com/Bernardoow/tibia-map/commit/f3104c45680a7f6232f9cc20c9ee1937169431df

It's works. But the same code I tried use to load [tile](https://github.com/tibiamaps/tibia-map/blob/8178bc02b4c84bb8b5e6bb4233c916b3bd1dd7a8/src/_js/map.js#L119) and it not works very well =/

I will study more about leftletjs. If you know some think who can help.
